developer/tutorials/articles/03-upgrading-code-to-liferay-7.2/01-intro.markdown

```diff
@@ -0,0 +1,254 @@
+# Upgrading Code to @product-ver@
+
+Upgrading to @product-ver@ involves migrating your installation and code (your
+custom apps) to the new version. You'll learn how to upgrade your code in this
+section.
+
+Here are the code upgrade steps:
+
+1.  [Upgrade Your Development Environment](/docs/tutorials/7-2/-/knowledge_base/t/upgrading-your-development-environment)
+
+    Legacy project environments should be upgraded to the latest version of
+    Liferay Workspace to ensure you leverage all available features.
+
+    1.  Set up Liferay Workspace
+
+        1.  Create New Liferay Workspace <!--commandId="create_new_liferay_workspace"-->
+
+            Create a new Liferay Workspace.
+
+        2.  Import Existing Liferay Workspace
+
+            If you do not have an existing 7.x workspace, go to the previous
+            step.
+
+    2.  [Configure Liferay Workspace Settings](/docs/tutorials/7-2/-/knowledge_base/t/upgrading-your-development-environment#configuring-liferay-workspace)
```

Comment:
Where do these links point to?  do these pages exist anywhere?

Reply:
This specific one points [here](https://github.com/gamerson/liferay-docs/blob/86fbc3be209ae32785c23eeb5101b9fb6c18ac24/developer/tutorials/articles/03-upgrading-code-to-liferay-7.2/02-upgrading-your-development-environment.markdown#configuring-liferay-workspace). These are links that point to docs in this PR. These will be changing though to match exact steps in tool. But these should provide good examples for testing.